src/components/Statistics/Statistics.tsx

```diff
@@ -378,51 +337,41 @@ const Statistics = () => {
   const percentAddressPossible = useMemo(() => {
     let nbTotal = 0;
     let nbTotalEligible = 0;
-    dataEligibilityTest &&
-      dataEligibilityTest.map((entry: any) => {
-        const [entryYear, entryMonth] = entry?.filters?.date?.split('-') || [
-          'YYYY',
-          'MM',
-        ];
-        const key =
-          entryYear === '2022' && (entryMonth - 1 === 5 || entryMonth - 1 === 6) //Mai et juin
-            ? 'nb_visits'
-            : 'nb_events';
-        nbTotal +=
-          getEntryValue(entry, 'Formulaire de test - Adresse Inéligible', key) +
-          getEntryValue(
-            entry,
-            'Formulaire de test - Carte - Adresse Inéligible',
-            key
-          ) +
-          getEntryValue(entry, 'Formulaire de test - Adresse Éligible', key) +
-          getEntryValue(
-            entry,
-            'Formulaire de test - Carte - Adresse Éligible',
-            key
-          );
-        nbTotalEligible +=
-          getEntryValue(entry, 'Formulaire de test - Adresse Éligible', key) +
-          getEntryValue(
-            entry,
-            'Formulaire de test - Carte - Adresse Éligible',
-            key
-          );
+    dataActions &&
+      dataActions.map((entry: any) => {
+        if (entry) {
+          nbTotal +=
+            entry['Formulaire de test - Adresse Inéligible'] +
+            entry['Formulaire de test - Adresse Éligible'] +
+            (entry['Formulaire de test - Carte - Adresse Inéligible']
+              ? entry['Formulaire de test - Carte - Adresse Inéligible']
+              : 0) +
+            (entry['Formulaire de test - Carte - Adresse Éligible']
+              ? entry['Formulaire de test - Carte - Adresse Éligible']
+              : 0);
+          nbTotalEligible +=
+            entry['Formulaire de test - Adresse Éligible'] +
+            (entry['Formulaire de test - Carte - Adresse Éligible']
+              ? entry['Formulaire de test - Carte - Adresse Éligible']
+              : 0);
+        }
```

Comment:
Attention, le code n'est pas strictement identique au précédent. Avant, si tout était undefined, on avait des additions de 0. Maintenant, on peut additionner des undefined / null par exemple. et ça donnera des NaN.

Est-ce que `entry['Formulaire de test - Adresse Inéligible']` est toujours défini ?

`entry['Formulaire de test - Carte - Adresse Inéligible'] ? entry['Formulaire de test - Carte - Adresse Inéligible'] : 0` peut être simplifié en `entry['Formulaire de test - Carte - Adresse Inéligible'] ?? 0`
Si tu fais cette modif, il y a d'autres endroits dans la PR également.

Reply:
D'ailleurs, il y a un `dataActions.map` mais on utilise pas le résultat, donc un `forEach` serait plus approprié.

Reply:
A priori oui le entry['Formulaire de test - Adresse Inéligible'] est toujours défini mais dans le doute j'ajoute un test tu as raison.
Pour le reste aussi !

Reply:
:+1: